src/services/widgets/widgets.svc.coffee

```diff
@@ -74,22 +74,52 @@ angular
         setting.initialize()
       return true
 
-    @updateWidgetSettings = (widget, needContentReload=true) ->
+    @updateWidgetSettings = (widget, needContentReload=true, ignoreReach=false) ->
       widget.isEditMode = false
-
       if _.isEmpty(widget.settings)
         $log.warn("ImpacWidgetsSvc: Tried to update widget: #{widget.id} with no settings", widget)
         return false
 
+      # Check if widget settings should be written to all same widgets and use as a default
+      changedGlobalSetting = _.find widget.settings, (setting)-> setting.reach == 'dashboard'
+
+      # Update all same widgets and dashboard settings if needed
+      if (changedGlobalSetting && !ignoreReach)
+        return _self.updateAllSameWidgets(widget, ImpacDashboardsSvc.getCurrentDashboard(), changedGlobalSetting)
+
       widget.isLoading = true if needContentReload
       meta = _.reduce(_.map(widget.settings, (set) -> set.toMetadata() ), (result, setMeta) -> angular.merge(result, setMeta))
-      
-      return _self.update(widget, { metadata: meta }).then(
+
+      _self.update(widget, { metadata: meta }).then(
         (updatedWidget) ->
           if needContentReload
             _self.show(updatedWidget).finally( -> updatedWidget.isLoading = false )
       )
 
+    # TODO: move logic in ImpacDashboardsSvc
+    # Sets setting for all widgets with same name
+    @updateAllSameWidgets = (widget, dashboard, settings) ->
+      # Find name of parameter
+      paramName = _.keys(settings.toMetadata())[0];
+
+      # Find widgets with same name
+      sameWidgets = _.filter dashboard.widgets, (wgt)-> wgt.name == widget.name
```

Comment:
"paramName" in our situation is not the name like "sales_stage_selection" or "status_selection", it is key of "params-picker" setting and metadata of the setting is 
{value: ["Hot", "Cold", "etc.."], reach: "dashboard"}. But we have "params-picker" in at least three widgets, "sales-cycle", "sales-leads" and "sales-opportunities", so when I find all widgets by "params-picker" parameter name I receive all of them, and write "{value: ["Hot", "Cold", "etc.."], reach: "dashboard"}" to all of them, but "sales-opportunities" and "sales-cycle", for example, have different lists of values, so I overwrite those widgets with incorrect values. 
I will try to add extra key for all these widgets and I think it will help to find only required widgets.